canonical-web-and-design/ubuntu.com
# Updated button in sub heading

## Done

Updated button in the sub heading to "watch the webinar" linking to “https://ubuntu.com/smartstart” with “https://ubuntu.com/engage/ubuntu-core-20-webinar”

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the changes against the [copy doc](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit?ts=602e6dca#) and accept the correct changes

Fixes [#3838](https://github.com/canonical-web-and-design/web-squad/issues/3838)
